src/SourceBuild/content/test/Microsoft.DotNet.Installer.Tests/LinuxInstallerTests.cs

```diff
@@ -0,0 +1,374 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using Microsoft.VisualStudio.TestPlatform.Utilities;
+using Newtonsoft.Json.Linq;
+using System;
+using System.Collections.Generic;
+using System.IO;
+using System.IO.Compression;
+using System.Linq;
+using System.Net.Http;
+using System.Runtime.Intrinsics.Arm;
+using System.Text;
+using System.Threading.Tasks;
+using System.Xml.Linq;
+using Xunit;
+using Xunit.Abstractions;
+using static System.Runtime.InteropServices.JavaScript.JSType;
+
+namespace Microsoft.DotNet.Installer.Tests;
+
+public class LinuxInstallerTests : IDisposable
+{
+    private readonly DockerHelper _dockerHelper;
+    private readonly string _tmpDir;
+    private readonly string _contextDir;
+
+    private readonly string[] RpmDistroImages =
```

Comment:
Why is this an array?  What is the thinking on how this would expand?

Reply:
We will likely add more images. Having it as array makes the future modification simpler.

Reply:
I'm changing this to `InlineData` in a `Theory`.

Reply:
Fixed with https://github.com/dotnet/sdk/commit/64fa1b8e6d800687ab49eae0b2a483e937a83e73